lib/logstash/inputs/snmp/mib.rb

```diff
@@ -157,11 +157,21 @@ def eval_mib_dic(filename)
         sub('MIB =', 'mib =')
 
       mib = nil
-      eval(mib_hash)
+      silence_warnings do
+        eval(mib_hash)
+      end
       mib
     rescue Exception => e
       # rescuing Exception class is important here to rescue SyntaxError from eval
       raise(SnmpMibError, "error parsing mib dic file: #{filename}, error: #{e.message}")
     end
+
+    def silence_warnings
+      warn_level = $VERBOSE
+      $VERBOSE = nil
```

Comment:
keep verbose same when debug is enabled, otherwise disable:

```suggestion
      $VERBOSE = logger.debug? ? warn_level : nil
```

Reply:
@robbavey how do you think?